crates/loader/src/local/assets.rs

```diff
@@ -130,37 +152,45 @@ fn collect_placement(
     }
 
     let walker = WalkDir::new(&abs);
-    let files = walker
+    let dir_entries = walker
```

Comment:
I feel like this block should no longer need to change except for adding the relative path to the FileMount constructor?  (Which, as mentioned elsewhere, I suspect it not needed - but even if it is...)

Reply:
Yes, removed it in `collect_placement`. Thank you.